person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_writer :id
+  attr_accessor :name, :age
+
+  def initialize(age, _name: 'Unknown', _parent_permission: true)
+    @id = Random.rand(10...100)
+    @name = name
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission == true
+  end
+
+  def say_age
+    print @age
+  end
```

Comment:
- [ ] [OPTIONAL] the function `say_age` is a function, I assume, you used to test your code. please make sure to clean up your code by removing unused pieces of code before sending it for review